CONTRIBUTING.md

```diff
@@ -63,32 +62,38 @@ Then you just add a line to every git commit message:
 
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
-If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+If you set your `user.name` and `user.email` git configs, you can sign your
+commit automatically with `git commit -s`.
 
 ## Contribution process
 
 ### Development
 
- - Create a personal fork of the project on GitHub.
- - Use **sycl** branch as baseline for your changes.
- - Prepare your patch (follow [LLVM coding standards](https://llvm.org/docs/CodingStandards.html)).
- - Build the project and run all tests (see [GetStartedWithSYCLCompiler.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md))
+- Create a personal fork of the project on GitHub
+- Use **sycl** branch as baseline for your changes
+- Prepare your patch (follow
+  [LLVM coding standards](https://llvm.org/docs/CodingStandards.html))
+- Build the project and run all tests (see
+[GetStartedWithSYCLCompiler.md](sycl/doc/GetStartedWithSYCLCompiler.md))
 
 ### Review and acceptance testing
 
- - Create a pull request for your changes following [Creating a pull request instructions](https://help.github.com/articles/creating-a-pull-request/).
- - When the pull request is created signed-off check is done.
-     - **check_pr** - verifies that the signed-off mark is added to each commit message.
- - Once the pull request is approved by an Intel representative, build and functional testing are started.
-     - **sycl-ubu-x64-pr** - runs all LIT tests on machine with OpenCL runtimes for CPU and GPU.
- - Approval is lost once the PR branch is updated. New approval and checks rerun are required.
- - Once approval is received and all checks pass, the pull request is ready to be merged.
+- Create a pull request for your changes following [Creating a pull request
+instructions](https://help.github.com/articles/creating-a-pull-request/)
+- When the pull request is created signed-off check is done
+  - **check_pr** - verifies that the signed-off mark is added to each commit
+  message
+- Once the pull request is approved by an Intel representative, build and
+functional testing are done
+- Approval is reset once the PR branch is updated
+  - New approval and checks rerun are required
+- Once PR is approved and all checks pass, the pull request is ready for merge
```

Comment:
I would prefer the following text here:

- CI will run signed-off check as soon as PR is created, see **check_pr** CI check for results
- CI will run build and functional testing check as soon as PR is approved by Intel representative
  - New approval is needed if PR was updated (e.g. during code review)
- Once PR is approved and all checks pass, the pull request is ready for merge

Reply:
Done.